Faizan_Akhtar/profile-page/assets/styles/profile-page.css

```diff
@@ -0,0 +1,11 @@
+.jumbotron{
+    background-image: linear-gradient(to right, #fe5858, #ee9617);  
+    /* colors - #2a2a72 #009ffd  */
```

Comment:
get rid of commented code in all style files, html files and JS files

Reply:
sure!
Commented code will be removed